src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -246,6 +252,22 @@ public void setExecuteIgnoredModules(boolean omit) {
         executeIgnoredModules = omit;
     }
 
+    /**
+     * Sets checker threads number.
+     * @param checkerThreadsNumber The checker threads number.
+     */
+    public void setCheckerThreadsNumber(int checkerThreadsNumber) {
```

Comment:
@soon, please create an issue to add properties to documentation later in order not to forget to do this